tests/effects/tdiagnostic_messages.nim

```diff
@@ -0,0 +1,32 @@
+discard """
+  errormsg: "\'a\' can have side effects"
```

Comment:
use
```
nimoutFull: true
nimout: '''
blah blah
```
same as in https://github.com/nim-lang/Nim/pull/18384
(and remove line, file, errormsg)